lib/happo/snapshot_comparer.rb

```diff
@@ -27,65 +32,89 @@ def compare!
       # work.
       return no_diff if @png_before == @png_after
```

Comment:
unrelated: what do you think about moving `no_diff` up to a frozen constant to improve perf?